lib/workers/repository/process/write.js

```diff
@@ -36,6 +38,14 @@ async function writeUpdates(config, packageFiles, allBranches) {
       return res;
     }
     prsRemaining -= res === 'pr-created' ? 1 : 0;
+    const changesList = ['pr-created', 'pr-edited', 'automerged'];
```

Comment:
What about the case where a branch is created but the PR is not? Wouldn't we be better to implement this in the `commitFilesToBranch` function in `git/storage.ts`?

Reply:
You are right it makes more sense moving it to storage.ts. And it looks cleaner.

Reply:
done